modules/setup-bias-monitoring.adoc

```diff
@@ -0,0 +1,55 @@
+:_module-type: PROCEDURE
+
+[id="setup-bias-monitoring_{context}"]
+= Before you begin
+
+== Cluster setup
```

Comment:
Add UI steps in addition to CLI steps, see: https://docs.redhat.com/en/documentation/openshift_container_platform/4.14/html/building_applications/projects#creating-a-project-using-the-web-console_projects